themes/admin/Sea_Green/css/sea_green.css

```
@@ -17739,26 +17752,26 @@ ol.breadcrumb.title-bar-breadcrumb a:active {
-.navbar .nav-item a:not(.dropdown-item):hover {
+.navbar .nav-item > a:not(.dropdown-item):hover {
   color: inherit;
   box-shadow: none;
   outline: none;
   border-radius: 2px;
   border: 1px solid rgba(123, 97, 255, 0.9);
 }
-.navbar .nav-item a:not(.dropdown-item):focus-visible {
+.navbar .nav-item > a:not(.dropdown-item):focus-visible {
   color: inherit;
   outline: none;
```

Stylelint (17.11.1)

[error] Duplicate property "outline" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)